src/views/PermissionSend.vue

```diff
@@ -286,8 +286,15 @@ export default {
     },
     feeInUsdValue () {
       const gas = BN(this.request.args[0].gas, 16)
-      const feePerGas = BN(this.fees[this.activeNetwork]?.[this.activeWalletId]?.[this.assetChain][this.selectedFee].fee).div(1e9)
```

Comment:
Hi, maybe we can add a default value if we can't get the .fee field inside this.fees

BR!

Reply:
Do you have any in mind, for example 21000?

Reply:
O no sorry for not add details, I mean when we want to access to `this.fees[this.activeNetwork]?.[this.activeWalletId]?.[this.assetChain][this.selectedFee].fee` maybe we can get null/undefined and then `BN` constructor can throw an error.